test/references/JSONLD_Web ICRISAT.txt

```diff
@@ -0,0 +1,15 @@
+register(DataDep(
+    "Phenotypic evaluation data of medium duration Pigeonpea advanced varieties trial",
+    """
+    	Dataset: Phenotypic evaluation data of medium duration Pigeonpea advanced varieties trial
+	Website: http://dataverse.icrisat.org/dataset.xhtml?persistentId=doi:10.21421/D2/ZS6XX1
+	Author: Sameer Kumar, CV, Anupama Hingane
+	License: <img src ="https://licensebuttons.net/l/by/4.0/88x31.png">
```

Comment:
Looks like stripping the HTML is also required on this field.

And since license is potentially long,
it should probably be always last, after things like _date_